app/assets/stylesheets/signin_form.scss

```diff
@@ -0,0 +1,31 @@
+a {
+  color: #A486B6;
+}
+
+a:hover {
+  color: #532E69;
+}
+
+body {
+  background-color: #fff;
```

Comment:
Please update this to be upcased, I can see you are using upcased color values...